get_together/settings.py

```diff
@@ -209,6 +210,26 @@
     messages.ERROR: "alert-danger",
 }
 
+THEME_CONFIG = {
+    "actions": {
+        "host_event": {
+            "label": _("Host a Get Together"),
+            "icon": "far fa-calendar-plus",
+        }
+    },
+    "text_labels": {
+        "host": _("host"),
+        "host_verb": _("moderate"),
+        "hosted_by": _("Hosted by"),
+        "change_host": _("Change Host"),
+        "change_host_for": _("Change host for:"),
+        "host_as_myself": _("Host this event as myself"),
+        "which_team_hosting": _("Which team is moderating this event"),
```

Comment:
Please change "moderating" to "hosting" so that the text matches the previous text by default.